src/test/java/gov/nj/innovation/customAwsIdp/lambda/GetSamlResponseHandlerTest.java

```diff
@@ -203,4 +207,23 @@ private void setupNiceGroupDescriptionExtractor() {
                 .when(() -> CognitoGroupDescriptionMetadataExtractor.extract(anyString(), anyString(), anyString()))
                 .thenReturn(new CognitoGroupDescriptionMetadata("https://test.com", "specialrole"));
     }
+
+    private static class MockSsmClient implements SsmClient {
+        @Override
+        public String serviceName() { return "MockService"; }
+        @Override
+        public void close() {}
+        @Override
+        public GetParameterResponse getParameter(GetParameterRequest getParameterRequest) {
+            return GetParameterResponse.builder()
+                    .parameter(Parameter.builder().name(getParameterRequest.name()).value("123").build())
+                    .build();
+        }
+    }
+
+    private GetSamlResponseHandler createNiceGetSamlResponseHandler() {
```

Comment:
I named it this way because creating a default `GetSamlResponseHandler` object would cause the tests to fail (since the default `SsmClient` is not mocked).

I chose this naming `createNice...` as an homage to some of the mocking library naming conventions that I have seen in Java, like [this one](https://easymock.org/api/org/easymock/EasyMock.html#createNiceMock-java.lang.Class-).